product_docs/docs/pem/9/considerations/pem_pgbouncer/configuring_pgBouncer.mdx

```diff
@@ -7,17 +7,20 @@ redirects:
     - /pem/latest/pem_pgbouncer/03_configuring_pgBouncer/
 ---
 
-You must configure PgBouncer to work with the PEM database server. This example runs PgBouncer as the enterprisedb system user and outlines the process of configuring pgBouncer version 1.9 or later.
+You must configure PgBouncer to work with the PEM database server. This example runs PgBouncer as the enterprisedb system user and outlines the process of configuring PgBouncer.
 
-1.  Open a terminal window and navigate to the pgBouncer directory.
+1.  Open a terminal window and navigate to the PgBouncer directory.
 
 2.  Change the owner of the `etc` directory for pgBouncer (where `pgbouncer.ini` resides) to `enterprisedb`, and change the directory permissions to `0700`:
 
     ```shell
-    $ chown enterprisedb:enterprisedb /etc/edb/pgbouncer1.9
-    $ chmod 0700 /etc/edb/pgbouncer1.9
+    $ chown enterprisedb:enterprisedb /etc/edb/pgbouncer<x.x>
+    $ chmod 0700 /etc/edb/pgbouncer<x.x>
     ```
 
+!!! Note
+    If you've installed the community version of PgBouncer using the PostgreSQL repo instead of installing EDB PgBouncer from the EDB repo, the PgBouncer directory doesn't include `edb` in the path. The path for the community version of PgBouncer is `/etc/pgbouncer<x.x>`.
+
 3.  Change the contents of the `pgbouncer.ini` or `edb-pgbouncer.ini` file:
```

Comment:
```suggestion
3.  Change the contents of the `edb-pgbouncer.ini` file:
```
The changes we describe apply to the edb-pgbouncer.ini file. I think it is a little confusing to show the changes for that file when we mention the other one too. We could add more content around the differences between in the config for the two versions of PgBouncer (paths and filenames), probably up front, or it is probably easier for the reader and the author to have two sets of instructions.